### PR TITLE
Fail types utility #33

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

File: src/Domain.LinnApps/Reports/BuildsSummaryReportService.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public IEnumerable<ResultsModel> GetBuildsSummaryReports(DateTime from, DateTime
                                                    })
                                   {
                                       ReportTitle = new NameModel(
-                                          week.Key.ToString("d", CultureInfo.CurrentCulture))
+                                          week.Key.Date.ToString("d", new CultureInfo("en-GB")))
```
Comment (atcampbell) [inline src/Domain.LinnApps/Reports/BuildsSummaryReportService.cs]: Should it be "o" for an iso string here?

File: src/Service.Host/client/src/components/boardFailTypes/BoardFailTypes.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        orderAscending: false,
+        currentPage: 0,
+        rowsPerPage: 10
+    });
+    const [rowsToDisplay, setRowsToDisplay] = useState([]);
+
+    const classes = useStyles();
+
+    useEffect(() => {
+        const compare = (field, orderAscending) => (a, b) => {
```
Comment (atcampbell) [inline src/Service.Host/client/src/components/boardFailTypes/BoardFailTypes.js]: I've put a couple of compare functions in the shared library that might work here

Comment (lewisrenfrew) [reply src/Service.Host/client/src/components/boardFailTypes/BoardFailTypes.js]: nice, but I actually just axed the sorting capability as i didn't need it and blindly copied and pasted this code

File: src/Domain.LinnApps/Reports/BuildsSummaryReportService.cs (hunk 1 above)
Comment (lewisrenfrew) [reply src/Domain.LinnApps/Reports/BuildsSummaryReportService.cs]: nah, this is just formatting it for display on the report, so just going with uk date format

**Pull request merged**